RecoMET/METFilters/python/metFilters_cff.py

```diff
@@ -95,39 +95,23 @@
 from RecoMET.METFilters.hfNoisyHitsFilter_cfi import *
 
 metFilters = cms.Sequence(
-   HBHENoiseFilterResultProducer *
-   HBHENoiseFilter *
-   primaryVertexFilter*
-#   HBHENoiseIsoFilter*
-#   HcalStripHaloFilter *
-   CSCTightHaloFilter *
-#   hcalLaserEventFilter *
-   #Various proposals for updated halo filters.
-   ##2015 proposals: 
-   #CSCTightHaloTrkMuUnvetoFilter *
-   #CSCTightHalo2015Filter *
-   ##2016 proposals
-   #globalTightHalo2016Filter*
-   #globalSuperTightHalo2016Filter*
-   EcalDeadCellTriggerPrimitiveFilter* 
-   ecalBadCalibFilter*
-#   *goodVertices * trackingFailureFilter *
-   eeBadScFilter*
-#   ecalLaserCorrFilter *
-#   trkPOGFilters
-   chargedHadronTrackResolutionFilter *
-   BadChargedCandidateFilter*
-   BadPFMuonFilter *
-   BadPFMuonDzFilter *
-   hfNoisyHitsFilter *
-   BadChargedCandidateSummer16Filter*
-   BadPFMuonSummer16Filter *
-   muonBadTrackFilter
+    goodVertices *
```

Comment:
Who needs `goodVertices` now? (It was supposed to be needed by the tracking failure filter, which is not in the list)

Reply:
The filter is still needed. It basically checks whether there's at least one good vertex is the event which help rejecting some noise.